.travis.yml

```diff
@@ -9,8 +9,6 @@ node_js:
 script:
   - npm test
   - npm run travis-demo
-  # print names of all sensitive variables present
-  - ./bin/has-env.js NPM
```

Comment:
try variable prefix that's for sure present

- ./bin/has-env.js TRAVIS

Reply:
Fixed